system/kube-monitoring-admin-k3s/Chart.yaml

```diff
@@ -26,7 +26,7 @@ dependencies:
     version: "^2.5"
   - name: oomkill-exporter
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.5.11
+    version: "^1.x"
```

Comment:
Wouldn't this break if a new version is pulled in, since no helm dep up is run?

Reply:
My understanding is that `helm dep up` will update the lock file with the newest version.
`helm diff` & `helm upgrade` should use the `Chart.lock` to lookup the dependency version, no?